table/server/common/src/main/java/alluxio/master/table/DatabaseInfo.java

```diff
@@ -26,7 +26,9 @@
 public class DatabaseInfo {
   @Nullable
   private final String mLocation;
+  @Nullable
   private final String mOwnerName;
+  @Nullable
   private final PrincipalType mOwnerType;
```

Comment:
should the default values in the constructor continue to be `""` and `PrincipalType.USER`? or should they be `null`? what is the difference between `""` and `null` for `mOwnerName`?

SImilar questions for the location and comment